packages/ocaml-variants/ocaml-variants.5.3.0+BER/opam

```diff
@@ -76,4 +76,4 @@ depopts: [
   "ocaml-option-nnp"
   "ocaml-option-nnpchecker"
 ]
-available: !(os = "macos" & arch = "arm64") & os != "win32" & arch != "arm32" & arch != "x86_32"
+available: os != "win32" & arch != "arm32" & arch != "x86_32"
```

Comment:
Orthogonal to this PR, but this:
```suggestion
conflicts: [
  "ocaml-option-32bit"
  "ocaml-option-bytecode-only"
]
available: os != "win32"
```
and removing them from the list above is more correct.

Reply:
The `arm32-` and `x86_32-` variants are [failing in CI](https://opam.ci.ocaml.org/github/ocaml/opam-repository/commit/20f0163b9f8982d875cef11f67e18b1029403ae4).  I wonder if it's due to this change.